packages/react-navigation-native/src/index.ts

```diff
@@ -142,6 +142,26 @@ export const useFunnel = createUseFunnel(({ id, initialState }) => {
         if (delta === -1) {
           if (navigation.canGoBack()) {
             navigation.goBack();
+          } else {
+            // FIXME: 미완성 코드. 관련 에러: https://github.com/toss/use-funnel/issues/77
+            // 이렇게 해도 네이티브 뒤로가기 시의 문제는 여전히 해결할 수 없음
+            if (params && navigation.getState().index === 0 && params.isOverlay) {
```

Comment:
```suggestion
            if (params && params.index > 0 && navigation.getState().index === 0 && params.isOverlay) {
```

안전하게 index가 0 이상일때 처리하면 좋을 것 같습니다.